model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -111,42 +111,35 @@ class OrtConfigurationTest : WordSpec({
         }
 
         "correctly prioritize the sources" {
-            val configFile = createTestConfig(
-                """
-                ort {
-                  scanner {
-                    storages {
-                      postgresStorage {
-                        url = "postgresql://your-postgresql-server:5444/your-database"
-                        schema = schema
-                        username = username
-                        password = password
-                      }
-                    }
-                  }
-                }
-                """.trimIndent()
-            )
-
-            val env = mapOf("ort.scanner.storages.postgresStorage.password" to "envPassword")
+            val configFile = File("src/test/assets/reference.conf")
+            val env = mapOf("ort.scanner.storages.postgres.password" to "envPassword")
 
             withEnvironment(env) {
                 val config = OrtConfiguration.load(
                     args = mapOf(
-                        "ort.scanner.storages.postgresStorage.schema" to "argsSchema",
-                        "ort.scanner.storages.postgresStorage.password" to "argsPassword",
-                        "other.property" to "someValue"
+                        "ort.scanner.storages.postgres.schema" to "argsSchema",
+                        "ort.scanner.storages.postgres.password" to "argsPassword",
+                        "other.property" to "someValue",
+                        "ort.scanner.options.ScanCode.compatibility.maxVersion" to "4.0",
+                        "ort.scanner.options.ScanCode.properties.key2" to "overriddenValue"
                     ),
                     configFile = configFile
                 )
 
                 config.scanner?.storages shouldNotBeNull {
-                    val postgresStorage = this["postgresStorage"]
+                    val postgresStorage = this["postgres"]
```

Comment:
Why that change?

Reply:
The test now operates on `reference.conf`, so that it has more options available. And in this file, the Postgres storage is named only _postgres_.